forge-gui/res/cardsfolder/upcoming/hedge_whisperer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Hedge Whisperer
+ManaCost:G
+Types:Creature Elf Druid Detective
+PT:0/3
+K:You may choose not to untap CARDNAME during your untap step.
+A:AB$ Pump | Cost$ 3 G T CollectEvidence<4> | ValidTgts$ Land.YouCtrl | SorcerySpeed$ True | TgtPrompt$ Select target land you control | RememberTargets$ True | AILogic$ ContinuousBonus | SpellDescription$ Target land you control becomes a 5/5 green Plant Boar creature with haste for as long as Hedge Whisperer remains tapped. It's still a land. Activate only as a sorcery. (To collect evidence 4, exile cards with total mana value 4 or greater from your graveyard.)
```

Comment:
#4610 adds support in AnimateEffect so you can use the Duration

Reply:
still some leftover params...

Reply:
I think it's ok now ?